src/Ownable.ts

```diff
@@ -0,0 +1,42 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
```

Comment:
VSC shows me an error here and in other places where we use `import type`. Getting rid of a `type` did the trick but not sure which approach is better/correct.

![obraz](https://user-images.githubusercontent.com/57687279/126626111-12b1c723-6b73-4f93-8be6-c11c99d837f8.png)

Reply:
Please make sure you're on [TypeScript 3.8](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8-beta/#type-only-imports-exports) or higher. It looks fine in my VSC.